app/dashboard/templates/payout_bounty.html

```diff
@@ -68,7 +68,7 @@ <h5>{% trans "Basic Payout." %}</h5>
                       </span>
                   </a>
                 </div>
-                <div class="w-100">
+                <!-- <div class="w-100">
```

Comment:
Any chance we can switch this to `{% comment %}`  and the closure to `{% endcomment %}`